### PR TITLE
Prepare tooling for v1 index generation

### PR DESCRIPTION
**Pull request by AndreiBarsan**
Brings forth many improvements to allow v1 indexing to run (add altitude to UTM, fix some path bugs, use the new sanitized poses, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ Package development, testing, and releasing is performed with `poetry`. If you j
  3. Develop away
     - run commands like `poetry run python -m pit30m.cli`
  4. Test with `poetry run pytest`
+    - Advanced command: `poetry run pytest -ff --new-first --quiet --color=yes --maxfail=3 -n 4`
+    - This command will run tests, then wait for new changes and test them automatically. Test execution will run in parallel thanks to the `-n 4` argument.
+    - The command lets you get feedback on whether your code change fixed or broke a particular test within seconds.
```
Comment (una-dinosauria) [inline README.md:66]: awesome

File: pit30m/data/log_reader.py [hunk 2]
```diff
@@ -242,13 +243,14 @@ def continuous_pose_dense(self) -> np.ndarray:
...
+
+        Pose times are UNIX seconds.
         """
         pose_data = []
-        # TODO(andrei): Document the timestamps carefully.
         for pose in self.raw_pose_data:
             pose_data.append(
                 (
-                    pose["capture_time"],
+                    gps_seconds_to_utc(pose["capture_time"]).timestamp(),
```
Comment (una-dinosauria) [inline pit30m/data/log_reader.py:253]:
```suggestion
                    # TODO(julieta) The overhead of this conversion might be very large at scale. Consider vectorizing
                    gps_seconds_to_utc(pose["capture_time"]).timestamp(),
```

File: pit30m/data/log_reader.py [hunk 3]
```diff
@@ -280,12 +282,11 @@ def map_relative_poses_dense(self) -> np.ndarray:
         # The data also has pose and velocity covariance information, but I have never used directly so I don't know
         # if it's well-calibrated.
         pose_data = []
-        # XXX(andrei): Document the timestamps carefully. Remember that GPS time, if applicable, can be confusing!
         for pose in self.raw_pose_data:
             # TODO(andrei): Custom, interpretable dtype!
             pose_data.append(
                 (
-                    pose["capture_time"],
+                    gps_seconds_to_utc(pose["capture_time"]).timestamp(),
```
Comment (una-dinosauria) [inline pit30m/data/log_reader.py:289]:
```suggestion
                    # TODO(julieta) consider vectorizing
                    gps_seconds_to_utc(pose["capture_time"]).timestamp(),
```

File: pit30m/data/log_reader.py (hunk 2 above)
Comment (una-dinosauria) [reply pit30m/data/log_reader.py:253]: Or maybe we could leave everything on GPS timestamps? Aren't they standard in robotics?

**Review by una-dinosauria — APPROVED** [review]
LGTM, just a minor question/comment about conversion from gps to unix timestamps

File: pit30m/data/log_reader.py (hunk 2 above)
Comment (AndreiBarsan) [reply pit30m/data/log_reader.py:298]: eh, I think they sort of are but I think they may be confusing to vision/ML people

File: README.md (hunk 1 above)
Comment (AndreiBarsan) [reply README.md:66]: :smile:

**Comment by AndreiBarsan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #50** <a href="https://app.graphite.dev/github/pr/pit30m/pit30m/50" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/pit30m/pit30m/50?utm_source=stack-comment).

**Pull request merged**